bolt-cli/src/cli.rs

```diff
@@ -131,6 +138,29 @@ pub struct SendCommand {
     pub devnet_sidecar_url: Option<Url>,
 }
 
+/// Command for registering validators in the Bolt network.
+#[derive(Debug, Clone, Parser)]
+pub struct RegisterCommand {
+    /// The chain for which the delegation message is intended.
+    #[clap(long, env = "CHAIN", default_value = "holesky")]
+    pub chain: Chain,
+
+    #[clap(long, env = "MAX_COMMITTED_GAS_LIMIT", default_value_t = 10_000_000)]
```

Comment:
I forgot to mention this but can you add doc comments to all the flags in this `RegisterCommand`?